gc/gc-base/build.gradle.kts

```diff
@@ -69,8 +73,10 @@ dependencies {
   testImplementation(project(":nessie-spark-extensions-base")) { testJarCapability() }
   testImplementation(project(":nessie-spark-extensions"))
```

Comment:
nit: Is it good idea to rename `nessie-spark-extensions` to `nessie-spark-3.1-extensions` (just to have a similar syntax as spark-3.2 module)? Now reader has to figure out which version of spark is `nessie-spark-extensions`